src/sys/windows/selector.rs

```diff
@@ -407,6 +406,7 @@ pub struct SelectorInner {
     cp: Arc<CompletionPort>,
     update_queue: Mutex<VecDeque<Arc<Mutex<SockState>>>>,
     afd_group: AfdGroup,
+    active_poll_count: AtomicUsize,
```

Comment:
Since this we can only have a single `poll` call at a time can you make this: `active_poll: AtomicBool`? And add a comment explaining why this is needed. Your commit message has all the needed details, but its nice to have it in the code as well.

Reply:
Done! Thanks for the suggestions.